lib/account/pages/edit_account/edit_birth_date_page.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:motis_mitfahr_app/util/locale_manager.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import '../../../util/big_button.dart';
+import '../../models/profile.dart';
+
+class EditBirthDatePage extends StatefulWidget {
+  final Profile profile;
+  const EditBirthDatePage(this.profile, {super.key});
+
+  @override
+  State<EditBirthDatePage> createState() => _EditBirthDatePageState();
+}
+
+class _EditBirthDatePageState extends State<EditBirthDatePage> {
+  final TextEditingController _controller = TextEditingController();
+  DateTime? _date;
+
+  @override
+  void initState() {
+    super.initState();
+    _date = widget.profile.birthDate;
+    _controller.text = _date == null ? '' : localeManager.formatDate(_date!);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(S.of(context).pageProfileEditBirthDateTitle),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 8, vertical: 10),
+        child: Column(
+          children: [
+            TextField(
```

Comment:
if we have the datepicker this should be read-only i guess

Reply:
yes, good point 👍